data-products/_example_2/metadata/00-specification.yml

```diff
@@ -0,0 +1,24 @@
+specification:
+  data-platform.justice.gov.uk/name: "Fake data product"
+  # A good title and description will help users find your data
+  data-platform.justice.gov.uk/title: "Fake Data Product"
+  # TODO, need to add more to the description
+  data-platform.justice.gov.uk/description: "This is a fake data product that contains information about ...."
+  # tags are an array of quoted strings to further aid discoverability - e.g. ["keyword1", "keyword2", "keyword3"]
+  data-platform.justice.gov.uk/product-tags: ["Test", "Fake Data Product"]
```

Comment:
Not required, but nice if you could add some real keyword tags. Then we can use your product to test the search when we build it!